.github/workflows/merge.yaml

```diff
@@ -158,7 +158,13 @@ jobs:
     - name: Upload package to PyPi
       run: |
         source venv/bin/activate
-        make publish
+        twine upload dist/* --username $PYPI_USERNAME --password $PYPI_TOKEN
```

Comment:
@sandcha I missed the latest change in Github Actions and PyPi, secrets are not passed on.